middlewares/hacker.middleware.js

```diff
@@ -523,6 +523,40 @@ async function updateHacker(req, res, next) {
     }
 }
 
+/**
+ * Updates a hacker that is specified by req.params.email, and then sets req.email
+ * to the email of the hacker, found in Account.
+ * @param {{params:{id: string}, body: *}} req
+ * @param {*} res
+ * @param {*} next
+ */
```

Comment:
You'll need to get the hacker from an email, see the route that gets a hacker details from an email and what middlewares it calls.